stable/jenkins/master-image/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jenkins:2.32.3
-RUN /usr/local/bin/install-plugins.sh kubernetes:0.11 workflow-aggregator:2.5 credentials-binding:1.10 git:3.1.0 \
+FROM jenkinsci/jenkins:2.46.1
```

Comment:
Are we open to switch up the master repo on this? Until we find/create a better solution (as discussed yesterday) to use the actual base image and then run this script after on 1st launch of the container, perhaps switching to a different automated build will help avoid delays from one person approval/push of image.  I'm happy to add you and others to have the ability to approve PRs to my (or other) GitHub repo which would automatically build/push the new Jenkins master image.